operating/manage-vulnerabilities/common-vuln-management-tasks.adoc

```diff
@@ -1,45 +1,82 @@
+Historically, {product-title-short} provided a view of vulnerabilities discovered in your system in the vulnerability management dashboard. The dashboard is deprecated in {product-title-short} 4.5 and will be removed in a future release. For more information about the dashboard, see xref:../../operating/manage-vulnerabilities/vulnerability-management-dashboard.adoc#vulnerability-management-dashboard[Using the vulnerability management dashboard].
 
-include::modules/identify-container-image-layer-that-introduces-vulnerabilities.adoc[leveloffset=+1]
+The *Vulnerability Management* -> *Workload CVEs* page provides information about vulnerabilities in applications running on clusters in your system. You can view vulnerability information across images and deployments. The *Workload CVEs* page provides advanced filtering capabilities, including the ability to view images and deployments with vulnerabilities and filter by image, deployment, namespace, cluster, CVE, component, and component source.
 
-include::modules/identify-dockerfile-line-component-cve.adoc[leveloffset=+1]
+//Viewing CVEs
+include::modules/vulnerability-management20-view-cve.adoc[leveloffset=+1]
+
+//Viewing vulns in nodes
+include::modules/identify-vulnerabilities-in-nodes-vm20.adoc[leveloffset=+1]
+include::modules/disable-identify-vulnerabilities-in-nodes.adoc[leveloffset=+2]
+.Additional resources
+* xref:../../operating/examine-images-for-vulnerabilities.adoc#supported-operating-systems_examine-images-for-vulnerabilities[Supported operating systems]
+
+//viewing vulns in platforms
+
+include::modules/vulnerability-management20-view-platform-cve.adoc[leveloffset=+1]
+
+[id="excluding-CVEs"]
+== Excluding CVEs
 
-include::modules/view-details-only-for-fixable-cves.adoc[leveloffset=+1]
+You can exclude or ignore CVEs in {product-title-short} by snoozing node and platform CVEs and deferring or marking node, platform, and image CVEs as false positives. You might want to exclude CVEs if you know that the CVE is a false positive or you have already taken steps to mitigate the CVE. Snoozed CVEs do not appear in vulnerability reports or trigger policy violations.
 
-include::modules/identify-operating-system-of-the-base-image.adoc[leveloffset=+1]
+You can snooze a CVE to globally ignore it for a specified period of time. Snoozing a CVE does not require approval.
 
-include::modules/identify-top-risky-objects.adoc[leveloffset=+1]
+[NOTE]
+====
+Snoozing node and platform CVEs requires enabling the ROX_VULN_MGMT_LEGACY_SNOOZE environment variable.
+====
 
-include::modules/top-risky-images-components.adoc[leveloffset=+1]
+Deferring or marking a CVE as a false positive is done through the exception management workflow. This workflow provides the ability to view pending, approved, and denied deferral and false positive requests. You can scope the CVE exception to a single image, all tags for a single image, or globally across all images.
```

Comment:
```suggestion
Deferring or marking a CVE as false positive is done through the exception management workflow. This workflow provides the ability to view pending, approved and rejected deferral and false positive requests. You can restrict the CVE exception to a single image, all tags for a single image or globally for all images.
```

Reply:
The tab is literally called "denied requests," not rejected. Also, "scope" is accepted language here.